components/molecules/BenefitCardHeaderSummary.js

```diff
@@ -0,0 +1,33 @@
+import propTypes from 'prop-types'
+import { SummaryTypes } from '../../objects/UniversalBenefit'
+
+export default function BenefitCardHeaderSummary(props) {
+  const t = props.locale
+  let typesWithLinks = [SummaryTypes.PaymentAmount, SummaryTypes.LatestStatus]
+
+  return (
+    <div>
+      <p>{t[props.summary.type].title}</p>
+      <p className="font-bold">
+        {props.summary.value}
+        <br />
+        {props.summary.status ?? null}
+      </p>
+      {!typesWithLinks.find((t) => t == props.summary.type) ? null : (
+        <p>
+          <a href={t[props.summary.type].link}>
```

Comment:
When using `<a>` tags that actually navigates to somewhere else in Next.js, we should try to always use the provided Next.JS `<Link>` component.

https://stackoverflow.com/questions/65086108/next-js-link-vs-router-push-vs-a-tag